homeassistant/components/hive/__init__.py

```diff
@@ -17,11 +17,16 @@
     async_dispatcher_connect,
     async_dispatcher_send,
 )
-from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity import Entity, EntityCategory
 from homeassistant.helpers.typing import ConfigType
 
 from .const import DOMAIN, PLATFORM_LOOKUP, PLATFORMS
 
+ENTITY_CATEGORY = {
+    "config": EntityCategory.CONFIG,
```

Comment:
Entity categories and platforms have a controlled relationship. Eg we only allow diagnostic category for read only entities like sensors and binary sensors. So this should be set in the platforms. I recommend doing this as part of entity descriptions.

Reply:
I’ll take a look at this